src/concurrent_log_handler/__init__.py

```diff
@@ -531,17 +533,6 @@ def shouldRollover(self, record):
         del record  # avoid pychecker warnings
         return self._shouldRollover()
 
-    def _shouldRollover(self):
```

Comment:
Actually from a second look, i think the `shouldRollover` method should be moved as well, not sure why my local test behaved as expected, seems like it should crash?

Reply:
Well, there is a general Exception handler it emit() that calls errorHandler() - which is defined in the Handler base class. So I'm thinking it may generate an error that gets logged to stderr through there, and wouldn't crash. But it might affect the behavior.

I'm pretty busy this month and moving next week, but I do plan to do some more testing and review here when I can find some time.